builtin/providers/rundeck/provider_test.go

```diff
@@ -0,0 +1,59 @@
+package rundeck
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+// To run these acceptance tests, you will need a Rundeck server.
+// An easy way to get one is to use Rundeck's "Anvils" demo, which includes a Vagrantfile
+// to get it running easily:
+//    https://github.com/rundeck/anvils-demo
+// The anvils demo ships with some example security policies that don't have enough access to
+// run the tests, so you need to either modify one of the stock users to have full access or
+// create a new user with such access.
+// Once you've got a user set up, put that user's API auth token in the RUNDECK_AUTH_TOKEN
+// environment variable, and put the URL of the Rundeck home page in the RUNDECK_URL variable.
```

Comment:
Thanks for these instructions! Can you fan out the "you'll need to modify one of the stock users to have full access" into steps so that a rundeck n00b can stumble their way through this? :grinning: